frontend/src/scenes/max/components/SidebarQuestionInputWithSuggestions.tsx

```diff
@@ -13,8 +15,14 @@ import { SidebarQuestionInput } from './SidebarQuestionInput'
 export function SidebarQuestionInputWithSuggestions(): JSX.Element {
     const { dataProcessingAccepted, activeSuggestionGroup } = useValues(maxLogic)
     const { setActiveGroup } = useActions(maxLogic)
+    const { coreMemory, coreMemoryLoading } = useValues(maxSettingsLogic)
     const { openSettingsPanel } = useActions(sidePanelSettingsLogic)
 
+    const tip =
```

Comment:
Nitpick: Could we render `/init` message by default for a split second? Like `afterMount` triggers the loader, `coreMemoryLoading` flips to `true`, and then `tip` switches to generic text, and then switches back again post-load if no memory exists?

Would it make sense to distinguish "not yet loaded" from "loaded but empty"?